include/ibm/management_console_rest.hpp

```diff
@@ -420,10 +420,12 @@ inline void
             asyncResp->res.result(
                 boost::beast::http::status::internal_server_error);
             asyncResp->res.jsonValue["Description"] = internalServerError;
-            BMCWEB_LOG_DEBUG << "deleteConfigFiles: Failed to delete the "
+            BMCWEB_LOG_ERROR << "deleteConfigFiles: Failed to delete the "
                                 "config files directory. ec : "
                              << ec;
         }
+        BMCWEB_LOG_CRITICAL
+            << "INFO:config files directory delete successful. PATH: " << loc;
```

Comment:
How much does this log?

Reply:
I am not sure if I  understand the question , 
if you are asking how many times we will get this log --> every time deletion of configfile is successsful we will get this log .